### DISCUSSION TITLE
Disable "New Version Available" prompt for Mirth Connect Administrator Launcher

### DISCUSSION THREAD

**Discussion by petegam** [nextgenhealthcare/connect#5831] [Feature/Enhancement Ideas]
Other than the user downloading and installing the new version, would it be possible to provide a way to disable the "New Version Available" prompt for Mirth Connect Administrator Launcher?  Thanks.

**Comment by tonygermano** [comment]
I see it as a nice nagging reminder to upgrade 😂 Is there a reason you don't want to be at the most recent version? I think at least 1.3.1 is necessary to run current mirth versions due to changes in nextgen's code signing cert.

**Comment by petegam** [comment]
As a vendor of Mirth, here is our Use Case:  We have a LOT of installations of Mirth.  We prefer to stay at the installed version until our dev team validates new versions of Mirth and new versions of the Launcher.  We also like to standardize on specific versions so we know which versions we are supporting in the field without having to troubleshoot N number of versions.

**Reply by pacmano1** [reply]
If you haven't already, submit a feature request in the features section.    

Perhaps ratther than a popup, some status bar at the bottom could serve as a reminder?   I suppose an option to just disable check entirely should exist but not defailt as "don't check".   As should there be a "check for update" button (so to speak).

**Reply by petegam** [reply]
Thank you for the suggestions.  I submitted feature request https://github.com/nextgenhealthcare/connect/issues/5836 ... including a workaround.
